app/controllers/api/comments_controller.rb

```diff
@@ -13,10 +14,12 @@ def create
 
   private
 
-  def update_and_set_user
+  def set_user
     @user = MobileUser.find_by(auth_token: @auth_token)
-    @user.update(name: params.fetch(:username))
-    @user
+  end
+
+  def update_username_if_changed
+    @user.update_attribute(:name, params.fetch(:username))
```

Comment:
`update_attribute` nie puszcza SQL-ki jeżeli parametr się nie zmienia? Jeżeli nie, to spoko, jeżeli tak, to chodziło mi raczej o coś takiego:

`@user.name = params.fetch(:username)`
`@user.save if @user.name_changed?` (metoda `name_changed?` powinno być dostępna automatycznie z modułu `ActiveModel::Dirty`